### ISSUE TITLE
SearchListControl: selected tags should be in a list

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-admin#3647]
The list of tags shown in the SearchListControl should be wrapped with `<ul>` and `<li>` tags, to make it easier to navigate them with a screen reader.

![imatge](https://user-images.githubusercontent.com/3616980/73457596-2db00880-4374-11ea-8c28-d7695a47d7e7.png)

The screenshot is from WC Blocks but the relevant code is in this repo: https://github.com/woocommerce/woocommerce-admin/blob/master/packages/components/src/search-list-control/index.js#L196
